src/libcore/num/f32.rs

```diff
@@ -153,6 +167,60 @@ pub mod consts {
 #[lang = "f32"]
 #[cfg(not(test))]
 impl f32 {
+    /// The radix or base of the internal representation of `f32`.
+    #[stable(feature = "assoc_int_consts", since = "1.42.0")]
+    pub const RADIX: u32 = 2;
+
+    /// Number of significant digits in base 2.
+    #[stable(feature = "assoc_int_consts", since = "1.42.0")]
+    pub const MANTISSA_DIGITS: u32 = 24;
+
+    /// Approximate number of significant digits in base 10.
+    #[stable(feature = "assoc_int_consts", since = "1.42.0")]
+    pub const DIGITS: u32 = 6;
+
+    /// [Machine epsilon] value for `f32`.
+    ///
+    /// This is the difference between `1.0` and the next larger representable number.
+    ///
+    /// [Machine epsilon]: https://en.wikipedia.org/wiki/Machine_epsilon
+    #[stable(feature = "assoc_int_consts", since = "1.42.0")]
+    pub const EPSILON: f32 = 1.19209290e-07_f32;
+
+    /// Smallest finite `f32` value.
+    #[stable(feature = "assoc_int_consts", since = "1.42.0")]
+    pub const MIN: f32 = -3.40282347e+38_f32;
+    /// Smallest positive normal `f32` value.
+    #[stable(feature = "assoc_int_consts", since = "1.42.0")]
+    pub const MIN_POSITIVE: f32 = 1.17549435e-38_f32;
```

Comment:
The non-associated constants should use the definition from the associated ones to have one source of truth.

Reply:
Done